flow-report/src/common.tsx

```diff
@@ -92,16 +92,22 @@ export const FlowStepThumbnail: FunctionComponent<{
     return <></>;
   }
 
-  if (lhr.gatherMode === 'timespan' && frames && frames.length) {
-    return <FlowStepAnimatedThumbnail frames={frames} width={width} height={height} />;
+  let thumbnail;
+  if (frames && frames.length) {
+    thumbnail = frames[frames.length - 1].data;
+    if (lhr.gatherMode === 'timespan') {
```

Comment:
nit: move to upper if statement?

Reply:
We would need to check `frames` twice then right?